pyobis/obisutils.py

```diff
@@ -1,6 +1,8 @@
 """
 Utility functions for internal use across various modules.
 """
+from urllib.parse import urlencode
+
 import requests
 
 obis_baseurl = "https://api.obis.org/v3/"
```

Comment:
My personal preference is to not have the last `/` there so the subsequent use of this string gets more readable. For example: `f"{obis_baseurl}/something"` vs`f"{obis_baseurl}something"`.

Reply:
Noted. I'll make sure code readability is one of my major priorities in subsequent works.